packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -63,7 +64,12 @@ export class MenuItem extends FASTElement {
      * @internal
      */
     public handleMenuItemKeyDown = (e: KeyboardEvent): boolean => {
-        this.change();
+        switch (e.keyCode) {
+            case keyCodeEnter:
+            case keyCodeSpace:
+                this.invoke();
+                return false;
+        }
 
         return true;
```

Comment:
unrelated to your change, but I am wondering why this is returning a boolean.

Reply:
I guess I could return void instead of false because both result in the event being default prevented, but in order to not preventDefault on the event we need to return true.   I found this surprising.